### PR TITLE
Feat/split apis

### PR DESCRIPTION
**Pull request by wdbasson**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -65,10 +99,31 @@ def create_app() -> FastAPI:
...
+    #     trust_registry,
+    #     verifier,
+    #     wallet_credentials,
+    #     wallet_dids,
+    #     webhooks,
+    #     sse,
+    # ]
+    if ROLE in ("governance", "tenant", "*"):
+        application = FastAPI(
+            root_path=ROOT_PATH,
```
Comment (rblaine95) [inline app/main.py:123]: https://fastapi.tiangolo.com/advanced/behind-a-proxy/

**Review by rblaine95 — APPROVED** [review]
For what my review is worth, LGTM

@ff137, any thoughts?

File: .github/workflows/continuous-deploy.yml [hunk 2]
```diff
@@ -326,6 +338,7 @@ jobs:
           repository: didx-xyz/charts
           token: ${{ secrets.PAT }}
           path: charts
+          ref: split-api
```
Comment (ff137) [inline .github/workflows/continuous-deploy.yml:341]: remember to update

File: shared/constants.py [hunk 3]
```diff
@@ -8,24 +8,37 @@
...
+)  # governance-trust-registry
+
 
 WEBHOOKS_URL = os.getenv("WEBHOOKS_URL", f"{url}:3010")
 WEBHOOKS_PUBSUB_URL = os.getenv("WEBHOOKS_PUBSUB_URL", f"ws://{host}:3010/pubsub")
 
 ACAPY_MULTITENANT_JWT_SECRET = os.getenv("ACAPY_MULTITENANT_JWT_SECRET", "jwtSecret")
 ACAPY_ENDORSER_ALIAS = os.getenv("ACAPY_ENDORSER_ALIAS", "endorser")
 
-CLOUDAPI_URL = os.getenv("CLOUDAPI_URL", f"{url}:8100")
```
Comment (ff137) [inline shared/constants.py:28]: the CLOUDAPI_URL env var is only used once and is identical to TENANT_FASTAPI_ENDPOINT. So I refactored that out

NB: CLOUDAPI_URL also removed in docker-compose and no longer needed in charts

**Review by ff137 — APPROVED** [review]
Nice 👍 
just included some renaming, deduplication, and removed the CLOUD_API env far (no longer needed)

**Pull request merged**